targets/TARGET_STM/TARGET_STM32L1/TARGET_MOTE_L152RC/objects.h

```diff
@@ -65,6 +65,7 @@ struct dac_s {
     PinName pin;
 };
 
+#define GPIO_IP_WITHOUT_BRR
 #include "common_objects.h"
```

Comment:
I don't have a MOTE_L152RC to test so I will have to trust you that this is required